docs/docs-content/byoos/usecases/vmware/konvoy.md

```diff
@@ -0,0 +1,465 @@
+  - [HashiCorp Packer](https://developer.hashicorp.com/packer/) version 1.10.1 or later.
+  - [tar](https://www.gnu.org/software/tar/) or similar tool to extract tarballs.
+  - [wget](https://www.gnu.org/software/wget/) or similar tool, such as curl to download files from the internet.
+
+- A valid [RHEL subscription](https://www.redhat.com/en/store/linux-platforms). You will need to provide the username
+  and password for the subscription during the build process.
+
+- VMware vSphere template for RHEL 8.8 or later. The Konvoy image builder will use this template to create the custom
+  RHEL with Konvoy image.
+
+<details>
+<summary>Click to learn more about VMware vSphere RHEL templates</summary>
+
+You can download RHEL 8.8 or later ISO files from the
+[Red Hat Developer Portal](https://developers.redhat.com/products/rhel/download?source=sso). The ISO can be uploaded to
+your vSphere Datastore and be used to deploy a VM. Deploy the VM and reference the ISO you uploaded to your Datastore as
+the input for the **CD/DVD Drive**.
+
+![A View of the ISO selection](/byoos_vmware_konvoy_iso-selection.png)
+
+After the VM is deployed, and you have completed the RHEL installation, verify you can SSH into the VM. If you can SSH
+and login to the VM, you can use it as a template for the Konvoy image builder. Save the VM as a template in vSphere.
+Reference the template in the Konvoy image builder configuration file.
+
+:::tip
+
+Configure the RHEL VM with the proper users and credentials before creating a VM template of it. Consider adding the
+following configuration to the `/etc/sudoers` file so that the root user and the `admin` and `sudo` groups can issue
+commands as root without entering a password.
+
+```shell
+root ALL=(ALL:ALL) ALL
+%admin ALL=(ALL) ALL
+%sudo ALL=(ALL) ALL
+Defaults        !authenticate
+#Defaults   !visiblepw
+```
+
+:::
+
+</details>
+
+- SSH credentials to the RHEL template you are specifying in the Konvoy image builder configuration file.
+
+  :::warning
+
+  Reach out to your VMware administrator if you need assistance with creating the VMware vSphere template. It's critical
+  you configure the RHEL template correctly to ensure the Konvoy image builder can the template to create the custom
+  RHEL with Konvoy image.
+
+  :::
+
+## Build Image
+
+The Konvoy image builder supports generating standard or FIPS-compliant Konvoy images. Select the appropriate tab for
+instructions on how to build the custom RHEL with Konvoy image.
+
+<Tabs>
+<TabItem label="Standard" value="non'fips">
+
+1.  Open a terminal session in your Linux VM and create a new directory for the Konvoy image builder project.
+
+    ```shell
+    mkdir --parents ~/builder
+    cd ~/builder
+    ```
+
+2.  Download the Konvoy builder release artifact from the GitHub releases page.
+
+    ```shell
+    wget https://github.com/mesosphere/konvoy-image-builder/releases/download/v2.8.5/konvoy-image-bundle-v2.8.5_linux_amd64.tar.gz
```

Comment:
Should we add a little comment about the versioning in this link? `We will be using version 2.8.5 in this guide`

Reply:
### Great suggestion